.github/workflows/macos-release.yml

```diff
@@ -106,9 +109,12 @@ jobs:
           name: test_results_xml
           path: ${{github.workspace}}/build/test-results/**/*.xml
 
+      - name: Package it
+        run: cpack
+
```

Comment:
Please note! We are now running CPack, which (hopefully) will produce the installer package(s).